src/main/resources/templates/login.html

```diff
@@ -18,27 +18,38 @@
                     <!--<p>&nbsp;</p>-->
                     <!--<img src="images/padlock1.png" width="127" height="300"/>-->
                 </div>
-                <div class="col-md-10">
-                    <div class="panel panel-default registration-panel">
-                        <div class="panel-heading ">
-                            <h3 class="dashhead" th:text="#{login.header}">Login</h3>
+                <p>&nbsp;</p>
+                <p>&nbsp;</p>
+                <div class="col-sm-2 col-sm-offset-2">
+                    <p>&nbsp;</p>
+                    <p>&nbsp;</p>
+
+                    <!--<img src="images/lightbulb.png" width="453" height="299" class="img-responsive"/>-->
+                </div>
+                <div class="col-md-5 col-sm-offset-1">
+                    <div class="panel panel-default registration-panel1">
+                        <div>
+                        <!--<div class="panel-heading ">-->
+                            <h3 class="dashhead loginpanel" th:text="#{login.header}">Log In</h3>
                         </div> <!-- panel-heading -->
+                        <div class="welcome">
+                        <p> <b>Welcome to ncl.sg </b></p>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <b> tag by <strong>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)